mlir_graphblas/algorithms.py

```diff
@@ -369,3 +368,143 @@ def vertex_nomination(
 
     node_of_interest = _vertex_nomination(graph, nodes_of_interest)
     return node_of_interest
+
+
+_pagerank = None
+
+
+def pagerank(
+    graph: MLIRSparseTensor, damping=0.85, tol=1e-6, *, maxiter=100
+) -> MLIRSparseTensor:
+    global _pagerank
+    if _pagerank is None:
+        csr64 = SparseEncodingType(["dense", "compressed"], [0, 1], 64, 64)
+        csc64 = SparseEncodingType(["dense", "compressed"], [1, 0], 64, 64)
+        csx64 = SparseEncodingType(["dense", "compressed"], None, 64, 64)
+        cv64 = SparseEncodingType(["compressed"], None, 64, 64)
+        aliases = AliasMap()
+        aliases["CSR64"] = csr64
+        aliases["CSC64"] = csc64
+        aliases["CSX64"] = csx64
+        aliases["CV64"] = cv64
+
```

Comment:
This pattern of regenerating aliases seems to repeat itself. Perhaps we should add a helper to abstract it out?

Reply:
I'll add a helper method for this